res/css/structures/_ViewSource.scss

```diff
@@ -34,8 +34,13 @@ limitations under the License.
     padding: 0.5em 1em 0.5em 1em;
     word-wrap: break-word;
     white-space: pre-wrap;
+    overflow-wrap: anywhere;
 }
 
 .mx_ViewSource_details {
     margin-top: 0.8em;
 }
+
+.mx_ViewSource_container {
+    margin-right: 1.125em;
+}
```

Comment:
Instead of this can we just use something like

![image](https://user-images.githubusercontent.com/2403652/162784748-73056c07-d1cb-4bd5-904d-ceb1718367cc.png)

1.125em isn't actually right and makes it asymetrical

![image](https://user-images.githubusercontent.com/2403652/162784859-4bf26be4-a15e-4a94-9861-bb61b04f1520.png)